src/test/java/org/owasp/wrongsecrets/challenges/docker/Challenge31Test.java

```diff
@@ -0,0 +1,42 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.spongycastle.util.encoders.Hex;
+
+@ExtendWith(MockitoExtension.class)
+class Challenge31Test {
+
+    private final String secretKey = new String(Hex.decode("534746325a53426849473570593255675a474635"));
+    @Mock
+    private ScoreCard scoreCard;
+
+    @Test
+    void spoilerShouldRevealAnswer() {
+        var challenge = new Challenge31(scoreCard);
+
+        Assertions.assertThat(challenge.spoiler()).isEqualTo(new Spoiler(secretKey));
+    }
+
+    @Test
+    void rightAnswerShouldSolveChallenge() {
```

Comment:
can you remove this test and the code on line 16 of the test? let's not give you rbeautiful obfuscated answer away this easily :D .

Reply:
if i remove line 16 then how can I give the spoiler

Reply:
can I inherit getanswer( ) mtd from the challenge31.java

Reply:
is this ok 
```
Assertions.assertThat(challenge.spoiler()).isEqualTo(new Spoiler(Challenge31.decode()));
```

Reply:
Note: i've not yet changed the method name in local yet